README.md

```diff
@@ -1 +1,59 @@
 # go-appr
+This package is a collection of `go-swagger` generated client bindings for `App Registry` which can be used to talk to `quay.io` or [appr](https://github.com/app-registry/appr) to pull down application packages.
+
+Swagger spec file has been obtained from here -  [appr-api-swagger.yaml](https://github.com/app-registry/appr/blob/master/Documentation/server/appr-api-swagger.yaml)
+
+
+## Generate Client Bindings
+[go-swagger](https://github.com/go-swagger/go-swagger) is a prerequisite before you can generate client bindings. 
+
+First, install `go-swagger`.
+```bash
+go get -u github.com/go-swagger/go-swagger/cmd/swagger
+```
+
+Next, run `go-swagger` to generate client bindings, as shown below.
+```bash
+# change directory to the root of the go-appr repo.
+swagger generate client --spec=./appr.spec.yaml --name=appregistry --api-package=appr --client-package=appregistry
+```
+
```

Comment:
Nit: it would help if you mentioned that you don't need to generate the client bindings to use the package.